controllers/webhook/webhook.go

```diff
@@ -101,12 +103,24 @@ func (w *OpenDataHubWebhook) checkDupCreation(ctx context.Context, req admission
 		fmt.Sprintf("Only one instance of %s object is allowed", req.Kind.Kind))
 }
 
+func (w *OpenDataHubWebhook) checkDeletion(ctx context.Context, req admission.Request) admission.Response {
+	if req.Kind.Kind != "DSCInitialization" {
```

Comment:
if we want to have a check here, we can do a  if  kind == DSC ,be more clear

Reply:
done